eido/conversion.py

```diff
@@ -76,41 +76,36 @@ def run_filter(prj, filter_name, verbose=True, plugin_kwargs=None):
     func = installed_plugins[filter_name]
 
     # run filter
-    conv_results = func(prj, **plugin_kwargs)
+    conv_result = func(prj, **plugin_kwargs)
 
     # if paths supplied, write to disk
     if paths is not None:
-        # make out dirs if they don't already exist
-        for path in paths:
-            if not os.path.exists(paths[path]):
-                os.makedirs(os.path.dirname(paths[path]), exist_ok=True)
-
-        # extract potential results
-        project_result = conv_results.get("project")
-        samples_result = conv_results.get("samples")
-
-        # write config conversion result
-        if project_result is not None:
-            with open(paths["cfg"], "w") as f:
-                f.write(project_result)
-
-        # write samples conversion result
-        if samples_result is not None:
-            with open(paths["samples"], "w") as f:
-                f.write(samples_result)
+        # map conversion result to the
+        # specified path
+        for result_key in conv_result:
+            result_path = paths.get(result_key)
+            if result_path is None:
+                _LOGGER.warn(
+                    f"Conversion plugin returned key that doesn't exist in specified paths: '{result_key}'. Printing to stdout."
+                )
+                sys.stdout.write(conv_result[result_key])
```

Comment:
we decided you should only print out if verbose, right? If not, just don't do anything with it.

Reply:
Yes. We can remove it then.